jsonschema-core.xml

```diff
@@ -324,10 +309,13 @@
 
         <section title="Schema references with $ref">
             <t>
-                In addition to a boolean value or an object using schema kewyords defined
-                in the meta-schema, a schema may be represnted by an object containing a "$ref" property.
-                The value of the $ref is a URI Reference.
+                The "$ref" keyword is used to reference an externally defined schema, and provides the ability to validate recursive structures through self-reference.
```

Comment:
The intention was to indicate schemas don't _necessarily_ have to be inside the current one; it wasn't intended to mean _exclusively_. I'll take this out.